mappings/community/default-requirements.json

```diff
@@ -2,7 +2,7 @@
   "id": "093c5272-5928-48c6-8eab-8f0ecf4c89a4",
   "request": {
     "method": "GET",
-    "urlPathPattern": "/offender/([A-Z0-9]*)/convictions/(?!1403337513)([0-9]*)/requirements"
+    "urlPathPattern": "/offender/([A-Z0-9]*)/convictions/(?!1403337513|636401162)([0-9]*)/requirements"
```

Comment:
The problem on CircleCI was that it was serving the default-requirements mock instead of the correct one.